elasticdl/docker/Dockerfile.dev

```diff
@@ -24,5 +24,10 @@ RUN pip install -r /requirements-dev.txt --extra-index-url=${EXTRA_PYPI_INDEX}
 # rest part of ElasticDL.  The generated data will be in /data.
 COPY elasticdl/python/data/recordio_gen/image_label.py /var/image_label.py
 RUN python /var/image_label.py --dataset mnist --fraction 0.25 \
-	--records_per_shard 4096 /data && \
-    rm -rf /root/.keras/datasets
+	--records_per_shard 4096 /data
+
+# Copy frappe dataset
+COPY elasticdl/python/data/recordio_gen/frappe_recordio_gen.py /var/frappe_recordio_gen.py
+RUN python /var/frappe_recordio_gen.py --data /root/.keras/datasets --output_dir /data/frappe \
+    --fraction 0.25
+RUN rm -rf /root/.keras/datasets
```

Comment:
Why removing this? I thought we added cache in travis.yml

Reply:
I just follow the former code here. If we remove the directory, the size of this image will be smaller. 

And for the travis, at the first time, the dataset will download inside docker. Since we mount the host directory to docker container, the dataset will also be write to host disk. Then, we cache the dataset in host. At the second time, the dataset will be used directly, which speed up the ci.

Reply:
Okay. Maybe add a comment here since otherwise it's hard to understand the logistics here